### ISSUE TITLE
data process

### ISSUE THREAD

**Issue by Yingying933** [xbh0720/LBKT#2]
Hello, can you process the training set and test set data into json file code in data2 folder open source code?

**Comment by xbh0720** [comment]
You can refer to this link https://github.com/bigdata-ustc/EduKTM/blob/main/examples/LBKT/data_preprocess.ipynb where I provide an example of processing ASSISTMENT2019 and the pytorch version of LBKT is also provided in the EduKTM repository.
